gen/src/rfcgen.act

```diff
@@ -898,12 +916,36 @@ def gen(stmts: dict[str, Stmt]) -> list[str]:
         stmt = stmts[stmt_name]
         arg_name = stmt.argument_name
         have_children = False
-        cattrs = []
+
+        # Sort the list of attributes so that attributes with cardinality 1 are
```

Comment:
It would be helpful to understand if lower or higher has higher priority, so 1 goes before what? 0? or something high?